mnamer/target.py

```diff
@@ -133,6 +133,10 @@ def parse(path: str, media: str) -> Metadata:
                     meta["series"] += " (%d)" % data["year"]
             if "season" in data:
                 meta["season"] = str(data["season"])
+                if meta["series"] in season_overrides:
+                    if str(meta["season"]) in season_overrides[meta["series"]]:
```

Comment:
It might help readability to use an intermediate variable for `str(meta["season"])` somewhere here.